src/components/templates/Playa/PlayaIcon.tsx

```diff
@@ -1,7 +1,6 @@
 import React from "react";
-import { PlayaIcon, Venue } from "types/Venue";
+import { AnyVenue, PlayaIcon, Venue } from "types/venues";
 import { WithId } from "utils/id";
-import { AnyVenue } from "types/Firestore";
```

Comment:
I thought Playa is completely removed from our codebase?

Reply:
See https://github.com/sparkletown/sparkle/pull/1080#discussion_r564114898